.patterns

```diff
@@ -0,0 +1 @@
+COMMITDATE [a-z0-9]{7}@[0-9]{6,8}
```

Comment:
Apparently in the docker project the patterns are not built into CLT. Since it works without them: 
```
 Manticore versions: 
- manticore: %{SEMVER} %{COMMITDATE} dev 
+ manticore: 6.3.3 88314e1fd@24071608 dev 
- columnar: %{SEMVER} %{COMMITDATE} 
+ columnar: 2.3.1 42f2b06@24070110 
- secondary: %{SEMVER} %{COMMITDATE} 
+ secondary: 2.3.1 42f2b06@24070110 
- knn: %{SEMVER} %{COMMITDATE} 
+ knn: 2.3.1 42f2b06@24070110 
```